data/redirects.yml

```diff
@@ -609,6 +609,8 @@
   - /go/build-cache-backends/
 "/build/exporters/":
   - /go/build-exporters/
+"/build/customize/bake/file-definition/":
+  - /build/bake/reference/
```

Comment:
@dvdksn Looks like the redirect does not work: https://docs.docker.com/build/customize/bake/file-definition/

Reply:
oh it's the wrong way around... 🤦🏻 now `/build/bake/reference/` is completely broken